app/models/project.rb

```diff
@@ -618,20 +618,16 @@ def check_status(deprecated_or_removed = false)
       update_attribute(:status, "Removed")
     elsif platform.downcase != "packagist" && [400, 404, 410].include?(response.response_code)
       update_attribute(:status, "Removed")
-    elsif platform.downcase == "clojars" && response.response_code == 404
```

Comment:
These conditions were unreachable, because they would have already been triggered by this:
`elsif platform.downcase != "packagist" && [400, 404, 410].include?(response.response_code)`